mvp-spec/ad-server-implementation.md

```diff
@@ -339,34 +320,26 @@ must be adapted to the existing API:
 <!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->
 ```json
 {
-    "transmissions": [
+    "version": "0.1",
+    "contents": [
         {
-            "version": 0,
-            "transaction_id": "a0651946-0f5b-482b-8cfc-eab3644d2743",
-            "receiver": "dsp1.com",
-            "status": "success",
-            "details": "",
-            "source": {
-                "domain": "dsp1.com",
-                "timestamp": 1639589531,
-                "signature": "12345_signature"
-            },
-            "children": []
+            "transaction_id": "f55a401d-e8bb-4de1-a3d2-fa95619393e8",
+            "content_id": "90141190-26fe-497c-acee-4d2b649c2112"
```

Comment:
What do you mean by sticking? Is there another place where we use camelCase? 

I saw only snake_case in the doc. At a technical level, do you see a significant positive impact ? As for me, it is (yet another) guideline on the subject where there is no standard. Fun fact: the JSON RFC uses uppercase for first letters. https://datatracker.ietf.org/doc/html/rfc8259

If you can provide a good rationale for it, it may be a good time because those changes is a significant shift in the specifications at a project level. However, I would suggest doing that in another review. This one is already significative.

Reply:
Sorry, by "sticking", I meant "use it as rule", but it is true that apart from `use_browsing_for_personalization`, `paf_browser_id` and `transaction_id`, there was no other example with "multi-word" key so no other camelCase example.

I guess my only rationale is that the direct translation of these properties into objects in C# or JS languages would feel awkward and would need to be adapted (these two languages usually favor camelCase). Also, the vast majority of JSON APIs I have worked with used this convention. But there is no standard, you are correct, it's a choice. 

Let's take that as a GitHub issue?

Reply:
https://github.com/criteo/prebid-addressability-framework-proposal/issues/118

Reply:
I think this approach is only meaningful for JS code. C# uses uppercase for first letters of properties (like the JSON CRF) so it doesn't really fit.